src/style.css

```diff
@@ -0,0 +1,17 @@
+@import "./styles/site.css";
+
+@import "./styles/buy.css";
+
+@import "./styles/review.css";
+
+@import "./styles/stars.css";
+
+@import "./styles/card.css";
+
+@import "./styles/price.css";
+
+@import "./styles/image.css";
+
+@import "./styles/code.css";
+
+@import "./styles/header.css";
```

Comment:
We just have to make 2 blocks according to Checklist point number 5.

"5.[BEM] - Don't make BEM structure too complicated. You don't need to create separate blocks for anything other than card itself and stars.